test/spectests/spectests.cpp

```diff
@@ -24,32 +24,33 @@ constexpr auto JsonExtension = ".json";
 constexpr auto UnnamedModule = "_unnamed";
 constexpr unsigned TestMemoryPagesLimit = (4 * 1024 * 1024 * 1024ULL) / fizzy::PageSize;  // 4 Gb
 
-// spectest module details:
-// https://github.com/WebAssembly/spec/issues/1111
-// https://github.com/WebAssembly/spec/blob/e3a3c12f51221daaa901691e674d7ff990029122/interpreter/host/spectest.ml#L33
-// https://github.com/WebAssembly/spec/blob/f85f6e6b70fff9ee0f6ba21ebcdad2183bd67874/test/harness/sync_index.js#L79
+// spectest module definition:
+// https://github.com/WebAssembly/spec/blob/master/interpreter/README.md#spectest-host-module
```

Comment:
Should we link to a commit hash or a tag? I think it is fine probably as is.

Reply:
Changed to commit hash